common/shell-local/config.go

```diff
@@ -53,6 +53,9 @@ type Config struct {
 	UseLinuxPathing bool `mapstructure:"use_linux_pathing"`
 
 	Ctx interpolate.Context
+
+	// internal use only; for the provisioner.
+	Winrmpassword string
```

Comment:
NIT: casing is incosistent with others.  I don't think we all agree on the correct casing, but I see `WinRMPassword` used elsewhere in this change.

Reply:
mmm excellent point. I actually meant to have this as lowercase entirely, to make sure that it was clear that it was internal-only and unexported.